Documentation/api-reference.md

```diff
@@ -100,13 +100,17 @@ See [MobileCore.resetIdentities](https://developer.adobe.com/client-sdks/documen
 
 Sends an Experience event to Edge Network.
 
+The sendEvent API has been improved to support Datastream overrides. This allows you to adjust your datastreams without the need for new ones or modifications to existing settings. The process involves two steps:
```

Comment:
rewording suggestion:
The sendEvent API has been improved to support Datastream overrides. -> Starting with vx.y.z the sentEvent API supports optional Datastream overrides.